dataclients/kubernetes/crd.go

```diff
@@ -0,0 +1,171 @@
+package kubernetes
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/zalando/skipper/eskip"
+)
+
+type routeGroupList struct {
+	Items []*routeGroupItem `json:"items"`
+}
+
+type routeGroupItem struct {
+	Metadata *metadata       `json:"metadata"`
+	Spec     *routeGroupSpec `json:"spec"`
+}
+
+type routeGroupSpec struct {
+	Hosts    []string          `json:"hosts,omitempty"`
+	Backends []*skipperBackend `json:"backends"`
+	Paths    []*pathSpec       `json:"paths"`
+}
+
+type pathSpec struct {
+	Path       string   `json:"path"`
+	Method     string   `json:"method,omitempty"`
+	Filters    []string `json:"filters,omitempty"`
+	Predicates []string `json:"predicates,omitempty"`
+	Backend    string   `json:"backend,omitempty"`
+}
+
+type algorithm int
```

Comment:
@aryszka Why we don't reuse these from the loadbalancer package?

Reply:
I am not against reusing them. One reason can be less coupling, but I think it's also fine to use them in this case.

Reply:
reusing